examples/azure-native-sample-project/pom.xml

```diff
@@ -13,7 +13,7 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <maven.compiler.release>11</maven.compiler.release>
-        <kotlin.version>1.7.20</kotlin.version>
+        <kotlin.version>1.8.0</kotlin.version>
```

Comment:
Why not `1.9.x`? :) 

(I have not researched it at all)

Reply:
I wanted to see that it will still work with 1.8.X. I think the Kotlin stdlib maintains compatibility for like one minor back? So it didn't work with 1.7.20 anymore, but I wanted to make sure 1.8.0 would still be fine